package-puzzles

```diff
@@ -95,7 +95,16 @@ if __name__ == '__main__':
         for points in sorted(puzzles_dict):
             puzzle = puzzles_dict[points]
             puzzledir = os.path.join(categoryname, 'content', mapping[points])
-            puzzlejson = puzzle.publish()
+            puzzledict = {
+                'author': puzzle.author,
+                'hashes': puzzle.hashes(),
+                'files': [f.name for f in puzzle.files if f.visible],
+                'body': puzzle.html_body(),
+            }
+            secretsdict = {
```

Comment:
I dropped that in there as a stub. I'm not sure how you want to handle secrets in package-puzzles so I just dropped this dict here assuming you'd tear it up and re-form it into whatever you want.